pediatric_rootlets/vertebrae_to_spinal_levels.py

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env python
+# -*- coding: utf-8
+# Compute distance between vertebral discs and PMJ along centerline
+#
+# For usage, type: python get_distance_pmj_dics -h
+
```

Comment:
```suggestion
# 
# The script requires the SCT conda environment to be activated (because we import the SCT's Image class):
#    source ${SCT_DIR}/python/etc/profile.d/conda.sh
#    conda activate venv_sct
#
```